spec/support/controller_macros.rb

```diff
@@ -26,6 +26,15 @@ def visit(arg)
     page.visit(arg)
 
     if RSpec.current_example.metadata[:js]
+      # When visiting a form we've already entered data into we will get the
+      # cached data prompt. We won't always, and we can't check for it. So we have
+      # to try and close the prompt every time and rescue if it errors.
+      begin
+        page.driver.browser.switch_to.alert
+        page.dismiss_prompt
+      rescue
+      end
```

Comment:
Can we omit this and use [implicit begin](https://github.com/rubocop-hq/ruby-style-guide#begin-implicit) instead?
```suggestion
      page.driver.browser.switch_to.alert
      page.dismiss_prompt
      rescue
      end
```

Reply:
I'll give that a try.

Ideally if we get the local caching changes merged, this can be removed entirely because the prompt won't exist anymore

Reply:
Ahhh what your suggesting is to add the rescue at the end of the method block, and that won't work. We need the method to continue execution regardless of this possible error.

It's ugly but we can remove it as soon as we merge the local caching stuff.

Reply:
I have removed this completely as the prompt no longer exists.